### ISSUE TITLE
how do you handle Decimal Data?

### ISSUE THREAD

**Issue by CharlesOkwuagwu** [xou/elixlsx#40]
Hi, great work on this library.

Please i'm using this to create excel files from a database.

Some of my data contains decimals (via the Elixir Decimal library, e.g. #Decimal<100.25>)

This error comes up:

```
ERR: %CaseClauseError{term: #Decimal<100.0>}
    (elixlsx) lib/elixlsx/xml_templates.ex:163: Elixlsx.XMLTemplates.get_content_type_value/2
    (elixlsx) lib/elixlsx/xml_templates.ex:197: anonymous fn/3 in Elixlsx.XMLTemplates.xl_sheet_cols/3
    (elixir) lib/enum.ex:1229: Enum."-map/2-lists^map/1-0-"/2
    (elixir) lib/enum.ex:1229: Enum."-map/2-lists^map/1-0-"/2
    (elixlsx) lib/elixlsx/xml_templates.ex:185: Elixlsx.XMLTemplates.xl_sheet_cols/3
    (elixlsx) lib/elixlsx/xml_templates.ex:258: anonymous fn/3 in Elixlsx.XMLTemplates.xl_sheet_rows/3
```

Please how may we use the Decimal type with `elixlsx`

Thanks.

**Comment by unti1x** [comment]
Hello. Elixir works only with primitive types, such as `string` or `float`. So, you have to use [Decimal.float/1](https://hexdocs.pm/decimal/Decimal.html#to_float/1) before using value with `elixlsx`.

**Comment by CharlesOkwuagwu** [comment]
Thanks. I have submitted a PR for that.

**Comment by unti1x** [comment]
Closed according to discussion in pr's #41, #42, #43
